WordPress/src/main/java/org/wordpress/android/ui/notifications/adapters/NotesAdapter.kt

```diff
@@ -359,29 +376,54 @@ class NotesAdapter(
         @Suppress("ForbiddenComment")
```

Comment:
Can we remove this suppression now?

Reply:
Sure, I can remove the suppression in my next PR